app-web/src/pages/index.js

```diff
@@ -181,13 +181,10 @@ export const Index = ({
     }),
   );
   const currentMeetups = allMeetups.filter(e => e.start.daysFromNow <= 0);*/
-  const eventsAndMeetups = currentEvents;
   if (results) {
-    results = removeUnwantedResults(results, allEvents, eventsAndMeetups);
+    results = removeUnwantedResults(results, allEvents, currentEvents);
```

Comment:
why change to currentEvents?

Reply:
Since we are not using Meetup, current Events make more sense, and eventsAndMeetups is not been used anywhere else, this way, we don't need to declare one more variable.